crypto/evp/evp_lib.c

```diff
@@ -1193,6 +1193,7 @@ int EVP_PKEY_CTX_get_group_name(EVP_PKEY_CTX *ctx, char *name, size_t namelen)
         return -1;
     return 1;
 }
+#endif  /* !FIPS_MODULE */
```

Comment:
Should we expose EVP_PKEY_Q_keygen() or just fix the calls?

Reply:
I went the path of least resistance: exposing stuff is easier than fixing things both in terms of coding and reviewing and we're on a deadline and I'm supposed to be doing other things...